.github/workflows/python-wheel.yml

```
@@ -112,6 +112,13 @@ jobs:
       - name: Wheel crate unit tests
         run: cargo test --locked -p streamlib-python-wheel --lib
 
+      # The tap controls decode transport-framed bags through the wheel's own
+      # decoder, so unlike the loopback fixture's analysis half they cannot ride
+      # the no-build job — they need the wheel this job installed above.
+      - name: Audio channel tap controls
+        working-directory: runtime/streamlib-engine/tests/fixtures
+        run: $GITHUB_WORKSPACE/sdk/streamlib-python-wheel/.venv/bin/python -m unittest test_tap_audio_channel
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:1: Double quote to prevent globbing and word splitting

(shellcheck)